packages/in_app_purchase/test/billing_client_wrappers/billing_client_wrapper_test.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:test/test.dart';
+import 'package:flutter/services.dart';
+
+import 'package:in_app_purchase/billing_client_wrappers.dart';
+import 'package:in_app_purchase/src/channel.dart';
+import '../fake_platform_views_controller.dart';
+
+void main() {
+  final FakePlatformViewsController fakePlatformViewsController =
+      FakePlatformViewsController();
+  BillingClient billingClient;
+
+  setUpAll(() {
+    Channel.override = SystemChannels.platform_views;
+    SystemChannels.platform_views.setMockMethodCallHandler(
```

Comment:
Why override the platform_views system channel? is it used by this plugin at all?

Can we not just set a mock handler for the `plugins.flutter.io/in_app_purchase` channel? and avoid the indirection added with the Channel class?